cumulusci/utils.py

```diff
@@ -135,10 +135,17 @@ def zip_inject_namespace(zip_src, namespace=None, managed=None, filename_token=N
     else:
         namespace = ''
 
+    # Handle tokens %%%NAMESPACED_ORG%%% and ___NAMESPACED_ORG___
+    namespaced_org_token = '%%%NAMESPACED_ORG%%%'
+    namespaced_org_file_token = '___NAMESPACED_ORG___'
```

Comment:
make overrideable in cumulusci.yml?

Reply:
I thought about that... for NAMESPACE I could see a use case where you want to inject multiple namespaces via different tasks.  However, for NAMESPACED_ORG, you'd only ever want one since an org can only ever have one namespace.